recipes/sagemaker-code-editor/build.sh

```diff
@@ -19,15 +68,24 @@ else
   PREFIX_DIR=$(pwd -P)
   popd
 fi
-
 # Go one level up
 PREFIX_DIR=$(dirname ${PREFIX_DIR})
-${PREFIX_DIR}/share/sagemaker-code-editor/bin/code-server-oss "$@"
+node "${PREFIX_DIR}/share/sagemaker-code-editor/out/server-main.js" "$@"
 EOF
-
 chmod +x ${PREFIX}/bin/sagemaker-code-editor
 
-if [[ "${build_platform}" == "${target_platform}" ]]; then
+# Replace bundled ripgrep with conda package
+mkdir -p ${PREFIX}/share/sagemaker-code-editor/node_modules/@vscode/ripgrep/bin
+cat <<EOF >${PREFIX}/share/sagemaker-code-editor/node_modules/@vscode/ripgrep/bin/rg
+#!/bin/bash
+exec "${PREFIX}/bin/rg" "\$@"
+EOF
+chmod +x ${PREFIX}/share/sagemaker-code-editor/node_modules/@vscode/ripgrep/bin/rg
+
+# Test sagemaker-code-editor and ripgrep
+if [[ "${CROSSCOMPILING_EMULATOR:-}" != "" ]]; then
+  ${PREFIX}/share/sagemaker-code-editor/node_modules/@vscode/ripgrep/bin/rg --help
```

Comment:
I think we should probably run test with conda tests rather than in the build think.

Also, why testing ripgrep which is a requirement?

Reply:
Overall, a lot seem to revolve around dropping the ripgrep node requirement, and use the conda package (which I salute).

Reply:
Context on ripgrep here - https://github.com/conda-forge/staged-recipes/pull/24163#discussion_r1464345707
Resolving this conversation since the related code is removed